### PR TITLE
adds views/urls for check uninstall

### PR DESCRIPTION
**Pull request by czpython**
Plan was to move this code from cmscloud api, figured might as well clean it up a bit.

### CODE REVIEW

File: aldryn_cms/urls.py [hunk 1]
```diff
@@ -3,8 +3,12 @@
 
 from cms.sitemaps import CMSSitemap
 
+from .views import check_uninstall_ok
+
 
 urlpatterns = [
+    # Not sure if we should hardcode admin path like this
+    url(r'^admin/~cmscloud-api/check-uninstall/$', check_uninstall_ok),
```
Comment (GaretJax) [inline aldryn_cms/urls.py:11]: Is that the same URL which is already used? I would prefer to have an aldryn specific prefix.
Is that resource protected in any way?

**Comment by GaretJax** [comment]
LGTM, but we should probably protect the view somehow

File: aldryn_cms/urls.py (hunk 1 above)
Comment (czpython) [inline aldryn_cms/urls.py:11]: url is the same, I'm not a fan of it but I think is needed given that we support both base projects

Comment (GaretJax) [inline aldryn_cms/urls.py:11]: Let's keep it then... if it's just this occurrence it's ok, otherwise we'll have to make something more configurable...

**Pull request merged**